packages/svelte-undp-design/src/lib/Loader/Loader.stories.ts

```diff
@@ -6,7 +6,19 @@ import Loader from './Loader.svelte';
 const meta = {
 	title: 'Example/Loader',
 	component: Loader,
-	tags: ['docsPage']
+	tags: ['docsPage'],
+	argTypes: {
+		size: {
+			type: 'string',
+			description: 'Size of the loader',
+			defaultValue: 'medium'
+		},
+		parameters: {
+			size: {
+				values: ['small', 'large']
```

Comment:
Could you also add `medium` size?

Reply:
1. Minimap
![image](https://user-images.githubusercontent.com/45137948/210767288-9b36be22-23f8-4797-a1c6-75f327fda112.png)
2. MapStyleCardList 
![image](https://user-images.githubusercontent.com/45137948/210767639-5145a95a-2201-4c4a-8c40-d8459e67c9e0.png)
3. TagFilter 
![image](https://user-images.githubusercontent.com/45137948/210768052-6be37a14-e6bd-483b-b080-f9a78ef4aa55.png)
4. MapQueryInfo 
![image](https://user-images.githubusercontent.com/45137948/210768607-66c37e64-23db-4c5d-90f6-c97ef8d0f491.png)